docs/reference/transfers.md

```diff
@@ -0,0 +1,266 @@
+
+Transfers `A` and `E` fail or succeed independently of `B`, `C`, and
+`D`.
+
+#### `flags.pending`
+
+A `pending` transfer reserves its [`amount`](#amount) in the
+debit/credit accounts'
+[`debits_pending`](./accounts.md#debits_pending)/[`credits_pending`](./accounts.md#credits_pending)
+fields respectively. The pending transfer is complete when the first of the
+following events occur:
+
+* If a corresponding
+  [`post_pending_transfer`](#flags.post_pending_transfer) is
+  committed, some or all of the pending transfer's reserved funds are
+  posted, and the remainder (if any) is restored to the original
+  accounts.
+* If a corresponding
+  [`void_pending_transfer`](#flags.void_pending_transfer) is
+  committed, the pending transfer's reserved funds are restored to
+  their original accounts.
+* If the pending transfer's timeout expires, the pending transfer's
+  reserved funds are restored to their original accounts. (When `flags.pending`
+  is set, the [`timeout`](#timeout) field must be non-zero.)
+
+#### `flags.post_pending_transfer`
+
+This flag causes the pending transfer (referred to by
+[`pending_id`](#pending_id)) to "post", transferring some or all of the money the
+pending transfer reserved to its destination, and restoring the
+remainder (if any) to its origin accounts. The `amount` of a
+`post_pending_transfer` must be less-than or equal-to the pending
+transfer's amount.
+
+* If the posted `amount` is 0, the full pending transfer's amount is
+  posted.
+* If the posted `amount` is nonzero, then only this amount is posted,
+  and the remainder is restored to its original accounts. It must be
+  less than or equal to the pending transfer's amount.
+
+Additionally, when this field is set:
+
+* `pending_id` must reference a pending transfer.
+* `flags.void_pending_transfer` must not be set.
+
+And the following fields may either be zero, otherwise must match the
+value of the pending transfer's field:
+
+* `debit_account_id`
+* `credit_account_id`
+* `ledger`
+* `code`
+
+#### `flags.void_pending_transfer`
+
+This flag causes the pending transfer (referred to by
+[`pending_id`](#pending_id)) to void. The pending transfer's
+referenced debit/credit accounts'
+[`debits_pending`](./accounts.md#debits_pending)/[`credits_pending`](./accounts.md#credits_pending)
+fields .
+
+Additionally, when this field is set:
+
+* `pending_id` must reference a pending transfer.
+* `flags.post_pending_transfer` must not be set.
+
+And the following fields may either be zero, otherwise must match the
+value of the pending transfer's field:
+
+* `debit_account_id`
+* `credit_account_id`
+* `ledger`
+* `code`
+
+### `amount`
+
+This is how much should be debited from the `debit_account_id` account
+and credited to the `credit_account_id` account.
+
+* Type is 64-bit unsigned integer (8 bytes)
+* Must not be zero, unless marking a pending transfer as posted
+
+### `timestamp`
+
+This is the time the account was created, as nanoseconds since
+UNIX epoch.
```

Comment:
The time of the "transfer".
I'd suggest rephrasing it with the same expression from `accounts.md`.

Reply:
Fixed, thank you!